alliance_auth/settings.py.example

```diff
@@ -166,6 +174,10 @@ STATIC_ROOT = '/home/allianceserver/allianceauth/static/'
 # This directory needs to be writable by the webserver
 EVEWHO_CACHE_DIR = '{}/cache'.format(BASE_DIR)
 
+# Cache directory for Fleet-Up, no ending /
+# by default this becomes ~/<installdir>/alliance_auth/cache
+# This directory needs to be writable by the webserver
+FLEETUP_CACHE_DIR = '{}/cache'.format(BASE_DIR)
```

Comment:
is this really required as we have removed the requests caching from auth as it causes issues with everything else . . . . . the eve who cache will be removed sooner or later as its just a dead setting